### ISSUE TITLE
Fedora postinstall script issue

### ISSUE THREAD

**Issue by nilgradisnik** [xou816/spot#64]
I'm trying to build this from Fedora 33 and I'm running into issues with `build-aux/meson/postinstall.py` script

```
Updating icon cache...
gtk-update-icon-cache: Failed to open file /usr/local/share/icons/hicolor/.icon-theme.cache : No such file or directory
Updating desktop database...
Compiling GSettings schemas...
Error opening directory “/usr/local/share/glib-2.0/schemas”: No such file or directory
```
It seems like the path should be just `/usr/share/` without `local` prefix. At least that's where I'm seeing icons and glib-2.0 contents on my system.

**Comment by xou816** [comment]
Hi! :)
Which prefix did you set for meson? It's only using /usr/local as the default prefix if MESON_INSTALL_PREFIX is undefined

**Comment by nilgradisnik** [comment]
You're right. I tried running this script manually because I figured there was a build step missing.

When I try to run it `./target/debug/spot` or `cargo run` I get an error:
```
thread 'main' panicked at 'Could not load resources: Error { domain: g-file-error-quark, code: 4, message: "Failed to open file “~/.local/share/spot/spot.gresource”: open() failed: No such file or directory" }', src/main.rs:55:10
```

README doesn't say anything about how to run it.

**Comment by xou816** [comment]
Hmm, how did you build it exactly?
If you use the same prefix as in the README, the spot.gresource file should have copied to ~/.local/share/spot indeed! Can you check that it's there?
